cmd/cuebe/cmd/apply.go

```diff
@@ -16,164 +16,128 @@ limitations under the License.
 package cmd
 
 import (
-	"errors"
 	"fmt"
-	"os"
-	"path"
+	"strings"
 
 	"cuelang.org/go/cue"
 	"cuelang.org/go/cue/load"
-	"github.com/loft-orbital/cuebe/cmd/cuebe/flag"
+	"github.com/loft-orbital/cuebe/cmd/cuebe/factory"
+	"github.com/loft-orbital/cuebe/cmd/cuebe/prompt"
 	"github.com/loft-orbital/cuebe/internal/utils"
 	"github.com/loft-orbital/cuebe/pkg/build"
-	bctxt "github.com/loft-orbital/cuebe/pkg/context"
 	"github.com/loft-orbital/cuebe/pkg/instance"
 	"github.com/loft-orbital/cuebe/pkg/manifest"
-	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 )
 
-type applyOpts struct {
-	*flag.BuildOpt
-	BuildContext *bctxt.Context
-	DryRun       bool
-	Force        bool
-}
-
 func newApplyCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:        "apply",
 		Aliases:    []string{"deploy"},
 		SuggestFor: []string{"install"},
-		Short:      "Apply release to Kubernetes",
-		Long: `Apply CUE release to Kubernetes.
+		Short:      "Apply context to k8s cluster.",
+		Long: `Apply context to k8s cluster.
 
-Apply uses server-side apply patch to apply the release.
+Apply uses server-side apply patch to apply the context.
 For more information about server-side apply see:
   https://kubernetes.io/docs/reference/using-api/server-side-apply/
+
+It applies every manifests found in the provided context,
+grouping them by instance if necessary.
 `,
 		Example: `
 # Apply current directory with an encrypted file override
-cuebe apply -i main.enc.yaml
+cuebe apply . main.enc.yaml
```

Comment:
Yeah it's true that the order of context is also the order of override if override there is. It can feel implicit but introducing a cli flag is not the best way to fix that. I'll try to document it more. In the end `context1` and `context2` are both valid contexts. There is no _injection_. 

E.g. with context1:
```txtar
-- main.cue --
package main

cm: {
        apiVersion: "v1"
        kind:       "ConfigMap"
        metadata: {
                name:      "potato"
                namespace: "default"
                labels: "cuebe.loft-orbital.com/instance": "cuebe-demo"
        }
        data: bestide: "vim"
}
```
and context2:
```txtar
-- secret.cue --
package main

secret: {
	apiVersion: "v1"
	kind:       "Secret"
	metadata: {
		name:      "tomato"
		namespace: "default"
		labels: "cuebe.loft-orbital.com/instance": "cuebe-demo"
	}
	data: bestide: bytes & 'vim'
}
```

No override is happening

Reply:
It's just a context can be a folder or a file or maybe something else. Cool if we can solve that with a bit more documentation.

Reply:
Indeed, I will improve the context doc then